site/content/contribute/server/style-guide.md

```diff
@@ -96,3 +96,17 @@ We prefer to use userID rather than userId. Same for abbreviations; HTTP is pref
 The name of a method's receiver should be a reflection of its identity; often a one or two letter abbreviation of its type suffices (such as "c" or "cl" for "Client"). Don't use generic names such as "me", "this" or "self", identifiers typical of object-oriented languages that give the method a special meaning.
 
 The above was just a short list of existing issues in the repo which should be rectified. In general, the broad list of rules are already available at [Effective Go](https://golang.org/doc/effective_go.html) and [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments.). They should be referred to whenever doing a code review.
+
+### Log levels
+
+The ultimate goal of logging is to provide observability. Hence, logs have a very important role for the application to communicate back to us in a healthy way. The most important features of effective communication is that it should be meaningful and concise. To achieve this, a log line should conform one of the definitions below.
+
+**Critical**: It represents the most urgent situations for the service to work properly and it is unable to meet even the minimum requirements. When a critical thing happens it is expected the service to stop.
+
+**Error**: Unlike “Critical”, in case of an error log, something important happened but it does not cause loss of the service. The administrator should investigate the incident. Hence it should provide enough information to start an investigation. An error can be a loss of service for a single user or something affecting a part of the application but does not break the service entirely.
```

Comment:
Might be useful to give some examples of what should be error and what should be warn. I feel like this is a common confusion.

Reply:
+1 to adding examples.